src/GitTools.Core/GitTools.Core.Shared/Exceptions/WarningException.cs

```diff
@@ -1,19 +1,25 @@
 namespace GitTools
 {
     using System;
+#if !NETSTANDARD
     using System.Runtime.Serialization;
+#endif
```

Comment:
Binary serialization has been deprecated in .net core. Most projects handle it by either completely removing [Serializable] or by using ifdefs, I've gone with the latter so nothing changes for classic .net.